cli/tests/integrations/test_analytics.py

```diff
@@ -115,64 +173,6 @@ def test_config_reporting_false():
         assert requests.post.call_count == 0
 
 
-@_mock
-def test_production_setting_true():
```

Comment:
Why does an OSError raise?  The tests should behave independently, so if they're not, we need to fix that.